index.html

```diff
@@ -45,6 +45,106 @@ <h1>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul class="tech">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>html</li>
+                <li>Javascript</li>
+              </ul>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul class="tech">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>html</li>
+                <li>Javascript</li>
+              </ul>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul class="tech">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>html</li>
+                <li>Javascript</li>
+              </ul>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul class="tech">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>html</li>
+                <li>Javascript</li>
+              </ul>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul class="tech">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>html</li>
+                <li>Javascript</li>
+              </ul>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about">
+        <!-- <img src="images/illustration-about-me-mobile.svg" alt="" /> -->
+        <div class="about-content">
+          <h2>About me</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="resume-btn">
+            <a href="#">Get My Resume</a>
+          </div>
+        </div>
+      </section>
```

Comment:
[OPTIONAL] 
Because the previous reviewer didn't require this specifically, I will mark it as Optional, but please take into consideration to refactor this bit of code, as you will need it in your next step.

- [ ] **Your project is not completed. You should add into the part of About-me section the following:**
![Screenshot from 2022-02-11 12-58-49](https://user-images.githubusercontent.com/83596135/153625351-2f41a457-d23b-4bb7-8d8f-25b4ed8113b9.png)

Please make sure to add this in your HTML file and style it with CSS accordingly with figma design